src/player-wrapper.js

```diff
@@ -266,7 +266,7 @@ PlayerWrapper.prototype.onReadyForPreroll = function() {
 PlayerWrapper.prototype.onPlayerReady = function() {
   this.h5Player =
       document.getElementById(
-          this.controller.getSettings().id).getElementsByClassName(
+          this.controller.getPlayerId()).getElementsByClassName(
```

Comment:
This is the same class - can't we just put this.getPlayerId() here and leave the controller out of it?

Reply:
Done.